polymath/extensions/vector_ops.py

```
@@ -58,8 +58,11 @@ def _mean_or_sum(arg, axis=None, *, recursive=True, _combine_as_mean=False):
     elif axis is None:
         if arg._shape:
             obj = Qube(func(arg._values[arg.antimask], axis=0), False, example=arg)
-        else:
-            obj = arg
+        else:  # pragma: no cover
+            # This is unreachable because if arg._shape is (), then the mask is boolean
+            # and either mask=False (line 50 hits) or mask=True (line 54 hits).
+            raise RuntimeError('This should be unreachable')
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)